Src/DependencyCollector/Shared/Implementation/DesktopDiagnosticSourceHttpProcessing.cs

```diff
@@ -0,0 +1,81 @@
+﻿#if NET45
+namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Web.Implementation;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the callbacks received from HttpDesktopDiagnosticSourceListener.
+    /// </summary>
+    internal sealed class DesktopDiagnosticSourceHttpProcessing : HttpProcessing
+    {
+        internal ObjectInstanceBasedOperationHolder TelemetryTable;
```

Comment:
Why make this internal? For tests? Should we make this a private field and expose an internal property instead? It's best practice to have fields private only - properties give more fine grained control on exposing what you want to expose to the outside world - for example may be you only want to give get permissions not set.